app/models/article.rb

```diff
@@ -277,6 +281,14 @@ def search_id
     "article_#{id}"
   end
 
+  def update_reading_list_reactions
+    if published
+      reactions.readinglist.find_each(&:index_to_elasticsearch)
```

Comment:
should these be inline? If an article is particularly popular this might be a lot to execute inline, for example [this post](https://dev.to/simonholdorf/9-projects-you-can-do-to-become-a-frontend-master-in-2020-n2h) is at the top of https://dev.to/top/infinity and it has 4615 reading list reactions

Reply:
never mind, this is the one in the backend :D